src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts working");

```

Comment:
The message 'Truck starts working' could be more informative. For example, it could specify what kind of work the Truck is doing.